packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.spec.ts

```diff
@@ -70,11 +70,12 @@ describe('parseHttpRequest', () => {
     expect(httpOperation).toMatchObject({
       id: '?http-operation-id?',
       method: 'post',
-      path: '/todos',
+      path: '/todos/{id}',
       servers: [{ url: 'http://test' }],
       request: {
         query: [{ name: 'limit', style: HttpParamStyles.Form, schema: { default: '10' } }],
         headers: [{ name: 'apikey', style: HttpParamStyles.Simple, schema: { default: '123' } }],
+        path: [{ name: 'id', style: HttpParamStyles.Simple, required: true }],
```

Comment:
hmm, maybe this is `type: string`, since it has to be in the url? 🤔

Reply:
Well, technically yes - but if we display `string` all the time, and there's a path param called `{id}` and it should be a `number`, we won't show that